xml/operations-configuring_check_plugins.xml

```diff
@@ -71,14 +71,12 @@
   The following Ansible example configures the Cassandra plugin and passes in
   related arguments.
  </para>
-<screen>- name: Run Monasca Agent detection plugin for Cassandra
-  monasca_agent_plugin:
-    name: "Cassandra"
-    args:
-      user: {{ FND_VDB.vars.monitor_user_name }}
-      password: {{ FND_VDB.vars.monitor_user_password }}
-      timeout: 10
-  no_log: True</screen>
+<screen>tasks:
```

Comment:
My cut and paste issue - you don't need to include the tasks: line.
Scott is going to look at the SampleCassandraPlugin code, so you might hold merging this until that change is also included.